### DISCUSSION TITLE
ae 编辑导出问题 drop shadow 为什么无法导出?

### DISCUSSION THREAD

**Discussion by herochengm** [Tencent/libpag#2386] [Q&A] [answered]
投影设置如下:
<img width="276" alt="img_v3_02d5_70457519-64c9-4053-aa24-f5cac4ebe52g" src="https://github.com/user-attachments/assets/25bba55f-97a4-491a-9a94-439ffeb0e9f5">

![img_v3_02d5_7c4c4862-d329-4d31-a9a3-9fe6dca9ee1g](https://github.com/user-attachments/assets/5cf5dc48-a884-40ee-b20e-0c745e0604a0)

还有glow 效果PAG view 预览有点奇怪:
<img width="498" alt="img_v3_02d5_192dccf3-9c05-4183-aba5-c30f159a5dfg" src="https://github.com/user-attachments/assets/9350d440-22f6-4617-adfc-96a169358507">

ae :24.1.0 (Build 78)
PAGViewer 4.3.6.102 (stable)

各位大仙求解.

**Comment by mosaictian** [comment, marked as answer]
你这个drop shadow是effects&Precents里的特效，PAG能支持的drop shadow是指Layer Style里的投影。
图层样式跟特效还是有差距，实现的方式不同。
